PED/SetPedCombatAttributes.md

```diff
@@ -22,96 +22,96 @@ enum CombatAttributes
 	BF_CanFlank = 42,
 	BF_SwitchToAdvanceIfCantFindCover = 43,
 	BF_SwitchToDefensiveIfInCover = 44,
-	_0xEB4786A0 = 45,
+	BF_ClearPrimaryDefensiveAreaWhenReached = 45,
 	BF_CanFightArmedPedsWhenNotArmed = 46,
-	_0xA08E9402 = 47,
-	_0x952EAD7D = 48,
+	BF_EnableTacticalPointsWhenDefensive = 47,
+	BF_DisableCoverArcAdjustments = 48,
 	BF_UseEnemyAccuracyScaling = 49,
 	BF_CanCharge = 50,
-	_0xDA8C2BD3 = 51,
-	_0x6562F017 = 52,
-	_0xA2C3D53B = 53,
+	BF_ClearAreaSetAdvanceIfDefensiveAreaReached = 51,
+	BF_UseVehicleAttack = 52,
+	BF_UseVehicleAttackIfVehicleHasMountedGuns = 53,
 	BF_AlwaysEquipBestWeapon = 54,
 	BF_CanSeeUnderwaterPeds = 55,
-	_0xF619486B = 56,
-	_0x61EB63A3 = 57,
+	BF_DisableAimAtAITargetsInHelis = 56,
+	BF_DisableSeekDueToLineOfSight = 57,
 	BF_DisableFleeFromCombat = 58,
-	_0x8976D12B = 59,
+	BF_DisableTargetChangesDuringVehiclePursuit = 59,
 	BF_CanThrowSmokeGrenade = 60,
 	BF_NonMissionPedsFleeFromThisPedUnlessArmed = 61,
-	_0x5452A10C = 62,
+	BF_ClearAreaSetDefensiveIfDefensiveCannotBeReached = 62,
 	BF_FleesFromInvincibleOpponents = 63,
 	BF_DisableBlockFromPursueDuringVehicleChase = 64,
 	BF_DisableSpinOutDuringVehicleChase = 65,
 	BF_DisableCruiseInFrontDuringBlockDuringVehicleChase = 66,
-	_0x0B404731 = 67,
+	BF_CanIgnoreBlockedLosWeighting = 67,
 	BF_DisableReactToBuddyShot = 68,
-	_0x7FFD6AEB = 69,
-	_0x51F4AEF8 = 70,
+	BF_PreferNavmeshDuringVehicleChase = 69,
+	BF_AllowedToAvoidOffroadDuringVehicleChase = 70,
 	BF_PermitChargeBeyondDefensiveArea = 71,
-	_0x63E0A8E2 = 72,
-	_0xDF974436 = 73,
-	_0x556C080B = 74,
-	_0xA4D50035 = 75,
+	BF_UseRocketsAgainstVehiclesOnly = 72,
+	BF_DisableTacticalPointsWithoutClearLos = 73,
+	BF_DisablePullAlongsideDuringVehicleChase = 74,
+	BF_DisableShoutTargetPosition = 75,
 	BF_SetDisableShoutTargetPositionOnCombatStart = 76,
 	BF_DisableRespondedToThreatBroadcast = 77,
-	_0xCBB01765 = 78,
-	_0x4F862ED4 = 79,
-	_0xEF9C7C40 = 80,
-	_0xE51B494F = 81,
-	_0x054D0199 = 82,
-	_0xD36BCE94 = 83,
-	_0xFB11F690 = 84,
-	_0xD208A9AD = 85,
+	BF_DisableAllRandomsFlee = 78,
+	BF_WillGenerateDeadPedSeenScriptEvents = 79,
+	BF_UseMaxSenseRangeWhenReceivingEvents = 80,
+	BF_RestrictInVehicleAimingToCurrentSide = 81,
+	BF_UseDefaultBlockedLosPositionAndDirection = 82,
+	BF_RequiresLosToAim = 83,
+	BF_CruiseAndBlockInVehicle = 84,
+	BF_PreferAirCombatWhenInAircraft = 85,
 	BF_AllowDogFighting = 86,
-	_0x07A6E531 = 87,
-	_0x34F9317B = 88,
-	_0x4240F5A9 = 89,
-	_0xEE129DBD = 90,
-	_0x053AEAD9 = 91, 
+	BF_PreferNonAircraftTargets = 87,
+	BF_PreferKnownTargetsWhenCombatClosestTarget = 88,
+	BF_ForceCheckAttackAngleForMountedGuns = 89,
+	BF_BlockFireForVehiclePassengerMountedGuns = 90,
+	MAX_COMBAT_FLAGS = 91,
 };
 ```
 
-The source for this enum is [here](https://alexguirre.github.io/rage-parser-dumps/dump.html?game=gta5&build=2944#_0x0E8E7201).
+The source for this enum is [here](https://alexguirre.github.io/rage-parser-dumps/dump.html?build=3095&game=gta5#CCombatData__BehaviourFlags).
```

Comment:
Since the enum is valid, I don't think having a external doc "confirming" is necessary.